web_viewer/client/package.json

```diff
@@ -5,15 +5,23 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "format": "prettier --write **/*.ts **/*.js"
   },
   "author": "Holger Rapp <hrapp@google.com>",
   "license": "Apache-2.0",
   "dependencies": {
+    "prettier": "^1.10.2",
     "@types/dat-gui": "0.6.1",
     "@types/three": "0.0.27",
     "dat-gui": "0.5.0",
     "three": "0.83.0",
     "typescript": "^2.2"
+  },
+  "prettier": {
+    "arrow-parens": "always",
+    "bracket-spacing": false,
```

Comment:
Why not `"bracket-spacing": true` here? That's the style we're now using internally.